store/store.go

```diff
@@ -1,10 +1,13 @@
 package store
 
-import "github.com/ProtonMail/gluon/imap"
+import (
+	"github.com/ProtonMail/gluon/imap"
+	"io"
+)
 
 type Store interface {
 	Get(messageID imap.InternalMessageID) ([]byte, error)
```

Comment:
Quick question, for symmetry, might we want to have streaming interface for both Get and Set? Might help with reducing copies down the line? Or not worth it you think?

Reply:
It currently doesn't have any benefits as we always need to decompress the whole thing in memory. There is a bit more work required to make sure that everything works correctly with the  RWLock system that sits over it.

I agree that it woud look better consistent, but it's not a high priority atm.